modules/gapi/src/backends/common/serialization.cpp

```diff
@@ -813,6 +845,16 @@ I::IStream& ByteMemoryInStream::operator>> (std::string& str) {
     }
     return *this;
 }
+I::IStream& ByteMemoryInStream::operator>> (std::map<std::string, std::string>& map_str) {
```

Comment:
Probably should be a template map in the future

Reply:
Let's do it now

Reply:
Done